sct.py

```diff
@@ -1434,6 +1435,9 @@ def create_test_release_jobs(branch, username, password, sct_branch, sct_repo):
         if group_name == 'raft':
             for jenkins_file in glob.glob(f'{base_path}/raft/*'):
                 server.create_pipeline_job(jenkins_file, group_name)
+        if group_name == 'tablets':
+            for jenkins_file in glob.glob(f'{base_path}/tablets/*'):
+                server.create_pipeline_job(jenkins_file, group_name)
```

Comment:
I think that this is going to change soon with @fruch's PR.

Reply:
Yes, I've discussed this with him. If his will be merged first, I'll rebase (Though, I can't seem to see any existing PR about it)

Reply:
this is the PR
https://github.com/scylladb/scylla-cluster-tests/pull/6866

Reply:
Clearly, I didn't go back far enough